### DISCUSSION TITLE
Not Found Plots

### DISCUSSION THREAD

**Discussion by sarjvela** [Chia-Network/chia-blockchain#5240] [Support]
I had intalled ubuntu and start plotting got arount 2tb but os hdd was very slow so I changed it with ssd also intalled ubuntu 20.04 restored wallet from 24 phrase, didnt move any plots or HDD's  inserted directory with plots but it says no plots found ... Restarted removed Directory fro GUI readded and stuff like that but no luck. please help 
![image](https://user-images.githubusercontent.com/30768061/118417294-7bb16280-b6c4-11eb-8c05-4a50d65b86a7.png)

**Comment by BrandtH22** [comment]
If you are still experiencing the issues it is recommended to upgrade to the latest version of chia. Please download it from here: https://www.chia.net/downloads/

Since this issue has been open for a number of years without additional comments, we will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions, comments, or if the issue persists after an update.

```
Please note that upgrading major versions of chia generally requires the config.yaml to be recreated using these steps:
1. Stop chia
2. Rename the config or move it to a different directory
3. In CLI run chia init
4. Copy any custom settings from the old config to the new (plot drive directories, pool info, trusted node info, etc) - be careful here as yaml is very picky about formatting including those leading spaces
5. Start Chia
```

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
